inginious/frontend/plugins/grader_generator/static/js/grader.js

```diff
@@ -1,4 +1,7 @@
-const isNotebook = $("#environment").val() === "Notebook";
+const environment =  $("#environment");
+const isNotebook = environment.val() === "Notebook";
+const isMultiLang = environment.val() === "multiple_languages";
+const isDataScience = environment.val() === "Data Science";
```

Comment:
It is probably more understandable if we set these const variables in the task_files_tab.js as they are not used right now in this plugin but in the UNCode plugin.